src/cli.ts

```
@@ -1003,6 +1032,62 @@ function getEnv(value: any): any {
+        if (key === "url") {
+            if (Object.keys(result[result.length - 1]).includes("url")) {
+                result.push({
+                    url: value,
+                });
+            } else {
+                result[result.length - 1].url = value;
+            }
+        }
+        if (key === "weight") {
+            assert(!("selectionWeight" in result[result.length - 1]), "duplicate weight option");
+            const parsedValue = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)